openhub/affiliated_committers.py

```diff
@@ -5,8 +5,14 @@
 
 
 def get_affiliated_committers_data(json_object):
+    """
+    :param json_object: json data of affiliated committers
+    :return: a list of affiliated committers dict
+    """
     data = json_object['response']['result'
                                    ]['affiliated_committers']['affiliator']
+    if isinstance(data, dict):
```

Comment:
This needs a comment, and you need to add an assert to verify the contents of the dict is what you expect: a single record.